clustoapi/util.py

```diff
@@ -115,7 +115,7 @@ def expanded():
         if isinstance(obj, clusto.drivers.resourcemanagers.ResourceManager):
             result['count'] = obj.count
 
-        if 'get_ips' in dir(obj) and not obj.entity.type == 'ipmanager':
+        if 'get_ips' in dir(obj) and 'manager' not in obj.entity.type:
```

Comment:
shouldn't this simply be a similar isinstance() check like above? i.e. if it has get_ips and is an instance of ResourceManager